framework/util/custom_vulkan_to_string.h

```diff
@@ -83,12 +83,6 @@ std::string ToString<VkLayerProperties>(const VkLayerProperties& obj,
                                         uint32_t                 tabCount,
                                         uint32_t                 tabSize);
 
-template <>
-std::string ToString<VkPerformanceCounterResultKHR>(const VkPerformanceCounterResultKHR& obj,
```

Comment:
Never called since no struct has a member of this type\* and it never appears in a function signature.  \*`VkPerformanceCounterResultKHR`